ui/src/app/cohort-search/selection-list/selection-list.component.tsx

```diff
@@ -335,22 +356,42 @@ export const SelectionList = withCurrentCohortCriteria()(class extends React.Com
     </React.Fragment> ;
   }
 
+  applyModifier(modifiers) {
+    if (modifiers) {
+      const modifierButtonText = '(' + modifiers.length + ')  MODIFIERS APPLIED';
+      this.setState({showModifiersSlide: false, modifierButtonText: modifierButtonText});
+    } else {
+      this.setState({showModifiersSlide: false, modifierButtonText: 'APPLY MODIFIERS'});
+    }
+
+  }
+
   render() {
     const {back, criteria} = this.props;
+    const {modifierButtonText, showModifiersSlide} = this.state;
     return <div>
-      <div style={{paddingTop: '0.5rem'}}>
-        {criteria && <div style={styles.selectionContainer}>
+        {!showModifiersSlide && criteria &&  <React.Fragment>
```

Comment:
I think we can simplify the conditions here, something like:
```
{!showModifiersSlide
  ? criteria && <React.Fragment>
      // selection list template
    </React.Fragment>
    <FlexRowWrap>
      // save & back buttons
    </FlexRowWrap>
  : <ModifierPage />
}
```

Reply:
Also, do we need to include criteria in the condition? I haven't tested this, but seems like it would always be true if we're already in the selection-list.

Reply:
Done.